sql/core/src/test/scala/org/apache/spark/sql/DatasetPrimitiveSuite.scala

```diff
@@ -170,23 +170,23 @@ class DatasetPrimitiveSuite extends QueryTest with SharedSparkSession {
   test("groupBy function, map") {
     val ds = Seq(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11).toDS()
     val grouped = ds.groupByKey(_ % 2)
-    val agged = grouped.mapGroups { (g, iter) =>
+    val aggregated = grouped.mapGroups { (g, iter) =>
```

Comment:
I don't feel strongly either way, but these don't necessarily need to be renamed. They're intentionally shorthand it seems. It might just reduce the number of lines touched.